package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "auth0-account-link-extension",
-  "version": "2.6.4",
+  "name": "auth0-account-link",
```

Comment:
Is there a reason for renaming? Not sure if it is used anywhere in install/deploy process.

Reply:
the build is not working if we do not fix naming.  the `a0-ext` tool adds .extension to the name, but the copy step does not expect that:
```auth0-account-link-extension.extension.2.6.5.js  65 KiB       0  [emitted]  main
Entrypoint main = auth0-account-link-extension.extension.2.6.5.js
 [0] ./webtask.js 758 bytes {0} [built]
 [1] external "auth0-extension-hapi-tools@1.3.1" 42 bytes {0} [built]
 [2] ./server/init.js 1.02 KiB {0} [built]
 [3] external "path" 42 bytes {0} [built]
 [4] external "auth0-extension-tools@1.4.0" 42 bytes {0} [built]
 [5] ./lib/config.js 106 bytes {0} [built]
 [6] ./server/index.js 2.08 KiB {0} [built]
 [7] external "@auth0/hapi@13.5.1" 42 bytes {0} [built]
 [8] external "inert@4.0.1" 42 bytes {0} [built]
 [9] external "hapi-auth-jwt2@7.0.1" 42 bytes {0} [built]
[10] ./lib/logger.js 303 bytes {0} [built]
[11] external "winston@1.0.0" 42 bytes {0} [built]
[12] ./server/routes.js 1.25 KiB {0} [built]
[30] ./lib/db.js 240 bytes {0} [built]
[71] ./server/handlers.js 701 bytes {0} [built]
    + 60 hidden modules
cp: ./dist/auth0-account-link.extension.2.6.5.js: No such file or directory
error Command failed with exit code 1.```

I can also fix the name used in the `cp` command, which will be less risky than changin the extension name (it might be used in unknown ways elsewhere)

Reply:
I pushed an update to package.json